test/network/test_conv1d.py

```diff
@@ -0,0 +1,180 @@
+### Toy example to test Conv1dConnection (the dataset used is MNIST but each image is raveled (each sample has shape (784,)).
+
```

Comment:
This code should be at MNIST examples. 
Test directory contain tests scripts that perform a integrity check for the models.